### DISCUSSION TITLE
The use of `joints_3d` is confusing for topdown pose estimation.

### DISCUSSION THREAD

**Discussion by pablovela5620** [open-mmlab/mmpose#801] [General]
This is relevant now that the repo performs both 2d and 3d keypoint estimations. When looking at 2d datasets such as the interhand_2d dataset and topdown pipeline transforms the naming scheme used is `joints_3d`, when what I'm understanding is meant is 2d joints + confidence/visibility? I think it would make sense to rename the use of `joints_3d` to `joints_2d` to reduce confusion.

**Comment by jin-s13** [comment]
No. joints_3d will represent the joints as [x,y,z].

**Comment by pablovela5620** [comment]
Yes this is what I mean, currently in the topdown configs that I'm looking at for 2d pose estimation they're not being used to represent [x,y,z] coordinates rather [u,v,c?]. Which is really confusing, if you look at both the topdown transforms as well as pretty much every other topdown dataset (so far I've only looked at the 2d hands one) they're using joints_3d rather then joints_2d. 

For example [here](https://github.com/open-mmlab/mmpose/blob/f1ec1113694ed824211ec1ba4a7e5ff43dcdb1aa/mmpose/datasets/datasets/hand/onehand10k_dataset.py#L93) its being labeled as joints_3d when I believe onehand10k is a 2d only dataset

**Comment by rubeea** [comment]
Hi @pablovela5620,
I was getting confused because of the same issue but when I printed the joints_3d array, I realized that for 2D joints the 3rd dimension is actually 0. So, it works ok.
